xcodeproj/internal/xcodeproj_macro.bzl

```diff
@@ -306,6 +308,28 @@ def xcodeproj(
             in the `top_level_targets` argument with a matching label will be
             excluded from the generated project. This overrides any targets
             specified in the `focused_targets` argument.
+        visionos_device_cpus: Optional. The value to use for `--visionos_cpus`
+            when building the transitive dependencies of the targets specified
+            in the `top_level_targets` argument with the `"device"`
+            `target_environment`.
+
+            **Warning:** Changing this value will affect the Starlark transition
+            hash of all transitive dependencies of the targets specified in the
+            `top_level_targets` argument with the `"device"`
+            `target_environment`, even if they aren't visionOS targets.
+        visionos_simulator_cpus: Optional. The value to use for `--visionos_cpus`
+            when building the transitive dependencies of the targets specified
+            in the `top_level_targets` argument with the `"simulator"`
+            `target_environment`.
+
+            If no value is specified, it defaults to the simulator cpu that goes
```

Comment:
There is only an arm64 simulator, so this should not have the defaulting logic like the others.